tldw_Server_API/app/core/Ingestion_Media_Processing/Audio/Audio_Transcription_Qwen3ASR.py

```
@@ -236,7 +236,7 @@ def _validate_model_path(model_path: str, allow_download: bool) -> Path:
         raise BadRequestError(
             f"Qwen3-ASR model path does not exist: {model_path}. "
             "Download the model first with: "
-            f"huggingface-cli download Qwen/Qwen3-ASR-1.7B --local-dir {model_path}"
+            f"hf download Qwen/Qwen3-ASR-1.7B --local-dir {model_path}"
         )
```

Ruff (0.15.6)

[warning] Avoid specifying long messages outside the exception class

(TRY003)